api.ts

```diff
@@ -2333,13 +2346,32 @@ export function makeSyncTable<
   }
 
   const responseHandler = generateObjectResponseHandler({schema: formulaSchema});
-  const execute = async function exec(params: ParamValues<ParamDefsT>, context: SyncExecutionContext) {
-    const {result, continuation} = (await wrappedExecute(params, context)) || {};
+  const execute = async function exec(
+    params: ParamValues<ParamDefsT>,
+    context: SyncExecutionContext,
+  ): Promise<SyncFormulaResult<K, L, SchemaT>> {
+    const syncResult = (await wrappedExecute(params, context)) || {};
     const appliedSchema = context.sync.schema;
+    const result = responseHandler({body: syncResult.result || [], status: 200, headers: {}}, appliedSchema) as Array<
+      ObjectSchemaDefinitionType<K, L, SchemaT>
+    >;
+    const {continuation, completion: syncExecutionCompletionMetadata} = syncResult;
+
+    if (continuation) {
+      if (syncExecutionCompletionMetadata) {
+        // eslint-disable-next-line no-console
+        console.warn('Ignoring syncExecutionCompletionMetadata because there is also a continuation.');
```

Comment:
I get a bit nervous about doing this kind of thing in the SDK because it gets bundled with the pack and we don't have any control over stopping it if we don't like it. Probably better to do this on the `coda` side if we even care to log it.

Reply:
Ok I'll do a followup PR

Reply:
https://github.com/coda/packs-sdk/pull/2970